.eslintrc.js

```diff
@@ -26,10 +31,13 @@ module.exports = {
         'no-trailing-spaces': ["error"],
         'one-var': ["error", "never"],
         'semi': ["error"],
-        'space-before-blocks': ["error"]
+        'space-before-blocks': ["error"],
+        // Disable these until we have converted the project to TS
```

Comment:
You could do overrides for TypeScript in order to allow the TS rules only for `.ts` files.